README.md

```diff
@@ -1,38 +1,64 @@
 # Slice for Visual Studio Code
 
-This language extension adds support for highlighting Slice files within Visual Studio Code.
-It supports highlighting the following file types:
+This extension provides support for the Slice Interface Definition Language (IDL).
+
+## Features
+
+- Syntax highlighting
+- Syntax validation
+- Error detection & reporting
+- 'Go to Definition' jumping
+
+### Syntax Highlighting and Validation
+
+This extension supports syntax highlighting and validation for the following Slice file types:
 
 - `.ice` files, which use the [original Slice](https://doc.zeroc.com/ice/latest/the-slice-language) syntax
 - `.slice` files, which use the [latest Slice](https://docs.icerpc.dev/slice2) syntax
 
-Additionally, this extension provides a Slice Language Server, for `.slice` files only.
+### Error Detection and Reporting
+
+Error checking is triggered every time a Slice file is saved or opened,
+and is only available for `.slice` files.
 
 ## Configuration
 
-The Slice Language Server can be configured using the following settings:
+The Slice language server that ships with this extension can be configured with the following settings:
+
+- `slice.languageServer.enabled`: A boolean indicating whether the server should be enabled. Defaults to `true`.
+
+- `slice.configurations`: An array of configuration sets (independent groups of Slice files). This allows multiple Slice projects to exist within a single repository. Each configuration set supports the following settings:
 
-- `slice.languageServer.enabled`: A boolean indicating whether the Slice Language Server should be enabled. Defaults
-  to `true`.
-- `slice.configurations`: An array of configuration sets. Configuration sets are independently compiled, allowing each
-  to define its own files and options for compilation.
-  - **Each configuration set is an object with the following properties:**
-  - `paths`: An array containing the paths to the target Slice reference files. Also supports directory paths.
-  - `addWellKnownTypes`: A boolean indicating whether to include the
-    [IceRpc well-known Slice files](https://github.com/icerpc/icerpc-slice/tree/main/WellKnownTypes) during compilation.
+    - `paths`: An array of paths to specify which Slice files should be included in this set.
+    This field is required.
+
+    - `addWellKnownTypes`: A boolean indicating whether to include the Slice definitions contained in the [IceRPC Slice](https://github.com/icerpc/icerpc-slice) repository.
+    These types are commonly used in applications utilizing Slice.
+    Defaults to `true`.
+
+If you do not specify any configuration sets, the extension will default to using the project's root directory for `paths`.
```

Comment:
I would use the present everywhere. No 'will'.

The extension defaults to ...
The language server only works with .slice files and ignores any ...

Reply:
I fixed the 2nd sentence, but left the 1st sentence as is.

In it, the `will` feels necessary, because the sentence is talking about a condition being met in the future:
`If you do not specify any configuration sets, the extension will default to ...`

For example:
`If you write bad code, the compiler will give an error` vs
`If you write bad code, the compiler gives an error`.
The 2nd one sounds wrong to me, especially compared to the 1st one.